circle.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
     - echo y | android update sdk --no-ui -a --filter extra-google-google_play_services
     - echo y | android update sdk --no-ui -a --filter extra-google-m2repository
     - echo y | android update sdk --no-ui -a --filter extra-android-m2repository
+    - sh templateChecks.sh
```

Comment:
Aside from this, you should also compile the template project, to make sure at least all breaking changes are addressed